scripts/Muon/GUI/Common/muon_group.py

```diff
@@ -137,6 +161,25 @@ def get_rebined_or_unbinned_version_of_workspace_if_it_exists(self, name):
 
         return None
 
+    def get_run_for_workspace(self, workspace_name):
+        for key, value in self._asymmetry_estimate.items():
```

Comment:
Can we use [`itertools.chain`](https://docs.python.org/3.6/library/itertools.html#itertools.chain) here to avoid the repitition?

```python
for key, value in itertools.chain(self._asymmetry_estimate.items(),
                                  self._counts_workspace.items(),
                                  self._asymmetry_estimate_rebin.items(),
                                  self._counts_workspace_rebin.items())
            if value.workspace_name == workspace_name:
                return key
```

Reply:
Yes good idea.